src/net/yapbam/gui/statementview/StatementViewPanel.java

```diff
@@ -604,10 +603,15 @@ public void actionPerformed(ActionEvent e) {
 	@SuppressWarnings("serial")
 	private JButton getBtnExport() {
 		if(btnExport == null) {
-			ExportComponent<ExporterParameters, FriendlyTable> exportC = new ExportComponent<ExporterParameters, FriendlyTable>() {
+			ExportComponent<StatementExporterParameters, FriendlyTable> exportC = new ExportComponent<StatementExporterParameters, FriendlyTable>() {
 				@Override
-				public Exporter<ExporterParameters, FriendlyTable> buildExporter() {
-					return new TableExporter() {
+				public Exporter<StatementExporterParameters, FriendlyTable> buildExporter() {
+					Statement statement = getStatementSelectionPanel().getSelectedStatement();
+					DecimalFormat ci = LocalizationData.getCurrencyInstance();
+					String statementId = statement.getId();

```

Comment:
I would prefer having LocalizationData.get("TransactionDialog.statement") before statement id rather than the statement id alone.